api-design-guidelines/index.md

```diff
@@ -613,6 +613,16 @@ is printed.
   var **radar**Detector: **Radar**Scanner
   var enjoys**Scuba**Diving = true
   ~~~
+
+  Words conventionally written with mixed case (usually proper names) should be
+  uniformly downcased when used in a lowercase context, and have only the
+  first letter upcased when used in a capitalized context:
+
+  ~~~swift
+  let **tex**Renderer = **TeX**Renderer()
+  let **ipad**Enabled = supports**IPad**()
```

Comment:
No, that is exactly what this convention is avoiding. Clearly it needs to be spelled out more explicitly.

(The intent was "only the first letter should be upcased; the rest should be left as conventionally written, neither upcased nor downcased". I need to rephrase the sentence to fit that in.)